docs/data/data-grid/selection/selection.md

```diff
@@ -84,25 +84,89 @@ The following demo shows the prop in action:
 
 {{"demo": "ControlledSelectionServerPaginationGrid.js", "bg": "inline"}}
 
-## apiRef [<span class="plan-pro"></span>](/x/introduction/licensing/#pro-plan)
+## Cell selection [<span class="plan-premium"></span>](/x/introduction/licensing/#premium-plan)
 
-The grid exposes a set of methods that enables all of these features using the imperative apiRef.
+The `DataGridPremium` comes with the ability to select rows enabled by default.
+You can also select cells by turning on the `cellSelection` prop.
 
-:::warning
-Only use this API as the last option. Give preference to the props to control the grid.
+```tsx
+<DataGridPremium cellSelection />
+```
+
+To select a single cell, click on it or, with the cell focused, press <kbd><kbd class="key">Shift</kbd>+<kbd class="key">Space</kbd></kbd>.
+Multiple cells can be selected by holding <kbd class="key">Ctrl</kbd> while clicking the cells.
+A cell can be deselected by also clicking it while <kbd class="key">Ctrl</kbd> is pressed.
+
+To select all cells within a range, you can use one of the shortcuts available:
+
+- Click on a cell, then drag the mouse over another cell and release it
+
+- Click on a cell and, while holding <kbd class="key">Shift</kbd>, click on another cell—if a third cell is clicked the selection will restart from the last clicked cell
+
+- Using the arrow keys, focus on a cell, then hold <kbd class="key">Shift</kbd> and navigate to another cell—if <kbd class="key">Shift</kbd> is released and pressed again, the selection will restart from the last focused cell
+
+The following demo allows to explore the cell selection feature.
+It has row selection disabled initially, but both features can be enabled at the same time.
+
+{{"demo": "CellSelectionGrid.js", "bg": "inline"}}
+
+### Controlled cell selection
+
+You can control which cells are selected with the `cellSelectionModel` prop.
+This props accepts an object whose keys are the row IDs that contain selected cells.
+The value of each key is another object, where the keys correpond to the `field` of the selected cell and the value is `true` to select the cell, or `false` to deselect it.
+Removing the field from the object also deselects the cell.
+
+```tsx
+// Selects the cell with field=name from row with id=1
+<DataGrid
+  cellSelectionModel={{ 1: { name: true } }}
+/>
+
+// Unselects the cell with field=name from row with id=1
+<DataGrid
+  cellSelectionModel={{ 1: { name: false } }}
+/>
+```
```

Comment:
I assume?

````suggestion
```tsx
// Selects the cell with field=name from row with id=1
<DataGridPremium
  cellSelectionModel={{ 1: { name: true } }}
/>

// Unselects the cell with field=name from row with id=1
<DataGridPremium
  cellSelectionModel={{ 1: { name: false } }}
/>
```
````